src/Resources/views/tailwind_2/_table.html.twig

```diff
@@ -295,7 +295,7 @@
                     {{ stimulus_target('araise/table-bundle/accordion', 'content') }}
                 >
                     <td class="w-12 px-3 py-3" aria-hidden="true"></td>
-                    <td colspan="{{ table.getColspan(tableActionsVisible|length > 0 ? 1 : 0) }}">
+                    <td colspan="{{ tableActionsVisible is defined and table.getColspan(tableActionsVisible|length > 0 ? 1 : 0) }}">
```

Comment:
NIT: easier syntax would be: `table.getColspan(tableActionsVisible|default([])|length > 0 ? 1 : 0)`